pyramid_nacl_session/utils.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from binascii import unhexlify
+
+from pyramid.session import JSONSerializer
+from pyramid.session import PickleSerializer
+
+from pyramid.settings import asbool
+
+from .session import EncryptedCookieSessionFactory
+
+
+def session_factory_from_settings(settings):
+    """
+    Return a Pyramid session factory using PyNaCl session settings
+    supplied from a Paste configuration file.
+    """
+
+    prefixes = ("session.", "pynacl.session.")
```

Comment:
I am not a huge fan of using a `session.` prefix without adding support for this to other session libraries or potentially to Pyramid directly. It may cause confusion because its not supported for anything else.

Reply:
`pyramid_beaker` supports `session.` too

Reply:
I'm a -0 on it, but I'll allow it.